[DOCS-8550] adding additional configs to network path static paths (#24341)
DataDog/documentation: content/en/network_monitoring/network_path/setup.md

## Overview

> Setting up Network Path involves configuring your Linux environment to monitor and trace the network routes between your services and endpoints. This helps identify bottlenecks, latency issues, and potential points of failure in your network infrastructure. Network Path allows you to configure either individual paths, or automatically discover network traffic paths depending on your needs.

Reviewer: ```suggestion
Setting up Network Path involves configuring your Linux environment to monitor and trace the network routes between your services and endpoints. This helps identify bottlenecks, latency issues, and potential points of failure in your network infrastructure. Network Path allows you to manually configure individual network paths or automatically discover them, depending on your needs.
```

## Setup

> ### Network Traffic Paths

Reviewer: ```suggestion
### Network traffic paths
```
Unless it's a proper noun. I see it's lowercase in the following paragraph, so this should be sentence case.

> Configure network traffic paths to allow the Agent to automatically discover and monitor network routes based on actual network traffic, without requiring you to specify endpoints manually.

Reviewer: ```suggestion
Configure network traffic paths to allow the Agent to automatically discover and monitor network paths based on actual network traffic, without requiring you to specify endpoints manually.
```
Feel free to ignore if routes is correct here. I just think we should be consistent with the term.

{{< tabs >}}
{{% tab "Dynamic paths" %}}

> Configure dynamic paths to allow the Agent to automatically discover and monitor network routes without requiring you to specify endpoints manually.

Reviewer 2: ```suggestion
Configure dynamic paths to allow the Agent to automatically discover and monitor network routes based on actual network traffic without requiring you to specify endpoints manually.
```

I'm wondering if it can help to add "based on actual network traffic" to let user know how those dynamic paths are created.
